x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_item.tsx

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLink, EuiListGroupItem } from '@elastic/eui';
+import { useNavigation } from '../../../lib/kibana';
+import { useGetSecuritySolutionLinkProps } from '../../links';
+import { SecurityPageName } from '../../../../../common/constants';
+
+export type NavItemCategories = Array<{ label: string; itemIds: string[] }>;
+export interface DefaultNavItem {
+  id: string;
+  label: string;
+  href: string;
+  onClick?: React.MouseEventHandler;
+  items?: PortalNavItem[];
+  categories?: NavItemCategories;
+}
+
+export interface CustomNavItem {
+  id: string;
+  render: () => React.ReactNode;
+}
+
+export type NavItem = DefaultNavItem | CustomNavItem;
+
+export interface PortalNavItem {
+  id: string;
+  label: string;
+  href: string;
+  onClick?: React.MouseEventHandler;
+  description?: string;
+}
+
+export const isCustomNavItem = (navItem: NavItem): navItem is CustomNavItem => 'render' in navItem;
+export const isDefaultNavItem = (navItem: NavItem): navItem is DefaultNavItem =>
+  !isCustomNavItem(navItem);
+
+export const useNavItems: () => NavItem[] = () => {
+  const getSecuritySolutionLinkProps = useGetSecuritySolutionLinkProps();
+  return [
+    {
+      id: SecurityPageName.dashboardsLanding,
+      label: 'Dashboards',
+      ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.dashboardsLanding }),
+      items: [
+        {
+          id: 'overview',
+          label: 'Overview',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.overview }),
+        },
+        {
+          id: 'detection_response',
+          label: 'Detection & Response',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.detectionAndResponse }),
+        },
+        // {
+        //   id: SecurityPageName.cloudPostureFindings,
```

Comment:
Is it for 8.3?

Reply:
I don't think so, this navigation component will be an MVP in 8.3, disabled by default. We can add new pages later, but I will confirm that assumption. Anyway, this nav_items file will be removed when the unified links config PR is merged.

Reply:
if you leave this in commented out, please put in a TODO note